tests/robustness/Makefile

```diff
@@ -75,6 +75,11 @@ test-robustness-issue19179: /tmp/etcd-v3.5.17-failpoints/bin
 	GO_TEST_FLAGS='-v -run=TestRobustnessRegression/Issue19179 -count 200 -failfast --bin-dir=/tmp/etcd-v3.5.17-failpoints/bin' $(TOPLEVEL_MAKE) test-robustness && \
 	  echo "Failed to reproduce" || echo "Successful reproduction"
 
+.PHONY: test-robustness-issue20693
```

Comment:
Please at the target to makefile

Reply:
Sorry, I am not following. 

Do you mean add it to README? :)

Reply:
No worries!

I have edited the README! 

I have tested against `3.5.0`, `3.4.23`, `3.5.23`, and all have broken watch guarantee, thus I put effected versions 3.4+. Note that I have to comment out `e2e.WithSnapshotCatchUpEntries(100)` when testing, as it wasn't supported.